rules-reviewed/quarkus/amazon/amazon-to-quarkus.windup.xml

```diff
@@ -0,0 +1,170 @@
+                    </project>
+                    <dependency groupId="software.amazon.awssdk" artifactId="kms"/>
+                </or>
+            </when>
+            <perform>
+                <hint title="Replace the 'software.amazon.awssdk:kms' dependency with Quarkus 'quarkus-amazon-kms' extension" effort="1" category-id="mandatory">
+                    <message>
+                        Replace the `software.amazon.awssdk:kms` dependency with the Quarkus dependency `io.quarkus:quarkus-amazon-kms`  
+                        Further information in the link below.
+                    </message>
+                    <link title="Quarkus - Guide" href="https://quarkus.io/guides/amazon-kms" />
+                </hint>
+            </perform>
+        </rule>
+        <rule id="quarkus-amazon-lambda-http-00000">
+            <when>
+                <or>
+                    <project>
+                        <artifact groupId="com.amazonaws.serverless" artifactId="aws-serverless-java-container-core"/>
+                    </project>
+                    <dependency groupId="com.amazonaws.serverless" artifactId="aws-serverless-java-container-core"/>
+                </or>
+            </when>
+            <perform>
+                <hint title="Replace the 'com.amazonaws.serverless:aws-serverless-java-container-core' dependency with Quarkus 'quarkus-amazon-lambda-http' extension" effort="1" category-id="mandatory">
+                    <message>
+                        Replace the `com.amazonaws.serverless:aws-serverless-java-container-core` dependency with the Quarkus dependency `io.quarkus:quarkus-amazon-lambda-http`  
+                        Further information in the link below.
+                    </message>
+                    <link title="Quarkus - Guide" href="https://quarkus.io/guides/amazon-lambda-http" />
+                </hint>
+            </perform>
+        </rule>
+        <rule id="quarkus-amazon-lambda-xray-00000">
+            <when>
+                <or>
+                    <project>
+                        <artifact groupId="com.amazonaws" artifactId="aws-xray-recorder-sdk-aws-sdk-v2"/>
+                    </project>
+                    <dependency groupId="com.amazonaws" artifactId="aws-xray-recorder-sdk-aws-sdk-v2"/>
+                </or>
+            </when>
+            <perform>
+                <hint title="Replace the 'com.amazonaws:aws-xray-recorder-sdk-aws-sdk-v2' dependency with Quarkus 'quarkus-amazon-lambda-xray' extension" effort="1" category-id="mandatory">
+                    <message>
+                        Replace the `com.amazonaws:aws-xray-recorder-sdk-aws-sdk-v2` dependency with the Quarkus dependency `io.quarkus:quarkus-amazon-lambda-xray`  
+                        Further information in the link below.
+                    </message>
+                    <link title="Quarkus - Guide" href="https://quarkus.io/guides/" />
```

Comment:
```suggestion
                    <link title="Quarkus - Guide" href="https://quarkus.io/guides/amazon-lambda#tracing-with-aws-xray-and-graalvm" />
```

Reply:
Solved in 08ce511 after https://github.com/quarkusio/quarkus/pull/13552 has been merged and released in Quarkus 1.10.2.Final